add docs for GADApplicationIdentifier (#4654)
expo/expo: packages/expo-ads-admob/README.md

Run `pod install` in the ios directory after installing the npm package.

> In your app's `Info.plist` file, add a `GADApplicationIdentifier` key with a string value of your AdMob app ID like the [Mobile Ads SDK iOS](https://developers.google.com/admob/ios/quick-start#update_your_infoplist)

Reviewer: ```suggestion
In your app's `Info.plist` file, add a `GADApplicationIdentifier` key with a string value of your AdMob app ID, as shown in Google's [Mobile Ads SDK iOS docs](https://developers.google.com/admob/ios/quick-start#update_your_infoplist).
```
